src/pl/plpgsql/src/pl_exec.c

```diff
@@ -240,6 +240,7 @@ static char *format_expr_params(PLpgSQL_execstate *estate,
 static char *format_preparedparamsdata(PLpgSQL_execstate *estate,
 						  const PreparedParamsData *ppd);
 
+bool orig_gp_enable_gpperfmon;
```

Comment:
Why is this variable in global scope? Do we need to change it outside this module?

Reply:
It's in global scope so that we can use it inside `plpgsql_exec_error_callback`